src/PostgREST/Observation.hs

```diff
@@ -61,12 +61,12 @@ type ObservationHandler = Observation -> IO ()
 
 observationMessage :: Observation -> Text
 observationMessage = \case
-  AdminStartObs port ->
-    "Admin server listening on port " <> show (fromIntegral (fromJust port) :: Integer)
+  AdminStartObs host port ->
+    "Admin server listening on " <> fromJust host <> ":" <> show (fromIntegral (fromJust port) :: Integer)
   AppStartObs ver ->
     "Starting PostgREST " <> T.decodeUtf8 ver <> "..."
-  AppServerPortObs port ->
-    "Listening on port " <> show port
+  AppServerPortObs host port ->
+    "Listening on " <> host <> ":" <> show port
```

Comment:
How about adding an io test for these? Example:

https://github.com/PostgREST/postgrest/blob/86a48b5b4152e4a5a92e60128cca4eb50d1230d2/test/io/test_io.py#L1235-L1245

Reply:
Done